apps/heft/src/plugins/TypeScriptPlugin/TypeScriptBuilder.ts

```diff
@@ -885,8 +887,45 @@ export class TypeScriptBuilder extends SubprocessRunnerBase<ITypeScriptBuilderCo
       outFolderPath = path.resolve(this._configuration.buildFolder, outFolderPath);
     }
 
+    outFolderPath = Path.convertToSlashes(outFolderPath);
+    outFolderPath = outFolderPath.replace(/(\/*)$/, '/'); // Ensure the outFolderPath ends with a slash
```

Comment:
Nit: `/\/*$/` will suffice. Don't need the capture.

Reply:
Good point.

Reply:
```suggestion
    outFolderPath = outFolderPath.replace(/\/*$/, '/'); // Ensure the outFolderPath ends with a slash
```